flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -985,6 +986,67 @@ int getNumScheduledTasks() {
 	 * restore from.
 	 * @param allowNonRestoredState Allow checkpoint state that cannot be mapped
 	 * to any job vertex in tasks.
+	 */
+	public boolean restoreLatestCheckpointedState(
```

Comment:
If you want, it seems to me like we can remove the return value from this method and change it to `void`.